### DISCUSSION TITLE
Record filtering not working !!

### DISCUSSION THREAD

**Discussion by chintannapster** [drivardxrm/LookupDropdown.PCF#4] [Q&A]
I downloaded the latest version 1.0.0.3 and try to apply it on two lookup fields having a relationship for filtering the record. but record filtering is not working. Can you please make it work or help me to to solve the issue ?
Thanks in advance !!

**Comment by drivardxrm** [comment]
Hi @chintannapster, 
Thanks for you interest ion the control.
are you able to share the setup for the lookup field that you are trying to filter ?
Ex. screenshot of the RelatedRecord filtering area + PCF control settings

**Comment by drivardxrm** [comment]
Hi @chintannapster ,
I haven't heard back from you, 
were you able to make the control work?

If no, please send me some context or screenshots so I can reproduce on my end.

I really want to understand what is not working so I can make a better control.

Thanks,

**Comment by drivardxrm** [comment]
Hi @chintannapster  
You might want to try the latest version v1.0.0.4
I fixed an issue on the related record filtering feature

Thanks for raising the issue
